### ISSUE TITLE
Update branding

### ISSUE THREAD

**Issue by kav2k** [theoldreader/safari#6]
Not sure if it's still maintained, but with the branding update the icon should be updated as well.

Feel free to pilfer the icons from the Chrome one: https://github.com/knyar/theoldreader-chrome/tree/master/source/img

**Comment by krasnoukhov** [comment]
Thanks for raising the point. Are you interested in maintaining it just in case?

**Comment by kav2k** [comment]
Ugh, doesn't it require an Apple Developer subscription? I don't have one.

**Comment by krasnoukhov** [comment]
Yeah I remember something like that. Well I don't seem to have settings access to this project anymore, so I can't invite you even if you wanted 😆 

Maybe @jfiorato can help with publishing the changes?

**Comment by jfiorato** [comment]
Yep, I'll take care of it. I imagine I'll have to get a new cert to sign the thing... We can maintain though. Thanks.
